draft-ietf-mls-protocol.md

```diff
@@ -3627,6 +3627,7 @@ Initial contents:
 | 0x0004          | MLS10_256_DHKEMX448_AES256GCM_SHA512_Ed448            | Y           | RFC XXXX  |
 | 0x0005          | MLS10_256_DHKEMP521_AES256GCM_SHA512_P521             | Y           | RFC XXXX  |
 | 0x0006          | MLS10_256_DHKEMX448_CHACHA20POLY1305_SHA512_Ed448     | Y           | RFC XXXX  |
+| 0x0007          | MLS10_256_DHKEMP384_AES256GCM_SHA384_P384.           | Y           | RFC XXXX  |
```

Comment:
```suggestion
| 0x0007          | MLS10_256_DHKEMP384_AES256GCM_SHA384_P384.            | Y           | RFC XXXX  |
```

Reply:
You committed to a one-line change! 
No problem - changes made, as well as a typo correction that was lingering in the spec.